src/main/java/org/broadinstitute/hellbender/tools/exome/GetHetCoverage.java

```diff
@@ -98,16 +99,19 @@
     )
     protected int minimumBaseQuality = 20;
 
+    @Argument(
+            doc = "Validation stringency for all BAM files read by this program.  Setting stringency to SILENT " +
+            "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
+            "do not otherwise need to be decoded.",
+            common=true)
+    protected ValidationStringency VALIDATION_STRINGENCY = ValidationStringency.DEFAULT_STRINGENCY;
+
     @Override
     protected Object doWork() {
-        if (pvalThreshold < 0 || pvalThreshold > 1) {
-            throw new UserException.BadArgumentValue(PVALUE_THRESHOLD_FULL_NAME,
-                    Double.toString(pvalThreshold),
-                    "p-value threshold should be in the [0, 1] range.");
-        }
-
-        final HetPulldownCalculator hetPulldown = new HetPulldownCalculator(REF_ARGUMENTS.getReferenceFile(), snpFile,
-                minimumMappingQuality, minimumBaseQuality);
+        SamReaderFactory.setDefaultValidationStringency(VALIDATION_STRINGENCY);
```

Comment:
This is kind of a big hammer to use since it sets global static state. See comments below.